### PR TITLE
[bunker] chore: add docstrings

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: src/services/bunker.py [hunk 1]
```diff
@@ -30,6 +39,12 @@ def is_bunker_mode(
         chain_config: ChainConfig,
         simulated_cl_rebase: LidoReportRebase,
     ) -> bool:
+        """
+        Triggers in 'OR' relation:
```
Comment (F4ever) [inline src/services/bunker.py:43]: Triggers if at least one of the following statements is True

File: src/services/bunker.py [hunk 2]
```diff
@@ -30,6 +39,12 @@ def is_bunker_mode(
...
+          - if any of the checks is True, then bunker mode is ON
+          - if all checks are False, then bunker mode is OFF
+        If it is the very first run, we don't check bunker mode
```
Comment (F4ever) [inline src/services/bunker.py:46]: Move to code comment

File: src/services/bunker.py (hunk 1 above)
Comment (F4ever) [inline src/services/bunker.py:43]: Check all three cases and one of those are True, bunker mode is On

File: src/services/bunker.py (hunk 2 above)
Comment (vgorkavenko) [reply src/services/bunker.py]: fixed

**Review by F4ever — APPROVED** [review]

**Pull request merged**